native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -376,7 +376,7 @@ class GameInputProxy {
             }
             case APP_CMD_DESTROY: {
                 CC_LOG_INFO("AndroidPlatform: APP_CMD_DESTROY");
-                IXRInterface *xr = _androidPlatform->getInterface<IXRInterface>();
+                IXRInterface *xr = CC_GET_XR_INTERFACE();
```

Comment:
I think it is better to use `CC_USE_XR` to enable xr related codes.

Reply:
We talked about this earlier, uniformly use IXRInterface to handle xr logic.
Use CC_USE_XR only in special cases.